packages/edit-site/src/components/page-switcher/index.js

```diff
@@ -111,8 +111,8 @@ export default function PageSwitcher( {
 			toggleProps={ {
 				children:
 					[ ...pages, ...categories, ...posts ].find(
-						( choice ) => choice.value === activePage.path
-					)?.label || activePage.path,
+						( choice ) => choice.value === activePage?.path
```

Comment:
**Non-blocking**:

What do we think about consolidating instances of `activePage?.path`? We could define `const activePagePath = activePage?.path` and replace R114, R115, etc. with references to the variable?

Reply:
sounds good!